Alexey-budgetCalc/modul.py

```diff
@@ -0,0 +1,211 @@
+import csv
+import matplotlib.pyplot as plt
+
+
+def month_input():
+    month = ""
+    flag = True
+    while flag:
+        month = input("Enter the month in format YYYY-MM \n")
+        if (len(month) == 7) and (month[4] == "-"):
+            if check_int(month[5:6]) and check_int(month[:3]):
+                flag = False
+    return month
+
+
+def date_input():
+    date = ""
+    flag = True
+    while flag:
+        date = input("Enter the date in format YYYY-MM-DD \n")
+        if (len(date) == 10) and (date[4] == "-") and (date[7] == "-"):
+            if check_int(date[8:9]) and check_int(date[5:6]) and check_int(date[:3]):
+                flag = False
+    return date
+
+
+def int_pos_input():
+    number = 0
+    while number < 1:
+        number = input("Input positive mumber\n")
+        try:
+            number = int(number)
+        except Exception:
+            print("Try again\n")
+            number = 0
+    return number
```

Comment:
You can also use regular expressions for matching integer values and convert them without error handling